src/App.jsx

```diff
@@ -13,46 +15,107 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const ALPHABETICAL = 'Alphabetical';
```

Comment:
without knowing the context it is difficult to understand why these variables
so I suggest renaming it to like `SORT_TYPE_BY_ALPHABETICAL`